fw/http.h

```diff
@@ -750,4 +754,32 @@ int tfw_http_resp_copy_encodings(TfwHttpResp *resp, TfwStr* dst,
 				 size_t max_len);
 void tfw_http_extract_request_authority(TfwHttpReq *req);
 
+static inline const BasicStr *
+tfw_http_method_id2str(int id)
```

Comment:
Need to use this method in all places where we need method string. For instance in access log.